docs: ✏️ add "Explore underlying data" user docs (#70807)
elastic/kibana: docs/drilldowns/explore-underlying-data.asciidoc

[[explore-underlying-data]]
== Explore the underlying data for a visualization

++++

> Explore the underlying data

Reviewer: ```suggestion
<titleabbrev>Explore the underlying data</titleabbrev>
```

++++

Dashboard panels have an *Explore underlying data* action that navigates you to *Discover*,

> where you can narrow your documents to the ones likely used in the visualization.

Reviewer: ```suggestion
where you can narrow your documents to the ones you'll most likely use in a  visualization.
```

[[explore-underlying-data]]

> == Explore data underlying visualizations

Reviewer 2: Maybe re-phrase to "Explore the underlying data powering your visualizations"?

The way it's phrased right now makes it confusing to figure out which noun the word "underlying" belongs to.

Reviewer: #### Title

How about this as a title:  Explore the underlying data for a visualization

#### Title abbreviation

Include this below the title, so a shorter version appears in the TOC:

++++
<titleabbrev>Explore the underlying data</titleabbrev>
++++

#### How to make the doc visible

In dashboard.asciidoc, after this line:

include::{kib-repo-dir}/drilldowns/drilldowns.asciidoc[]

add this line:

include::{kib-repo-dir}/drilldowns/explore-underlying-data.asciidoc[]

Author: Done.

> Dashboard panels have "Explore underlying data" actions that will redirect

Reviewer: Suggest combing the first two paragraphs:

> Dashboard panels have an **Explore underlying data** action that navigates you to Discover,
> where you can narrow your documents to the ones you use most often.
> This action is available for visualizations backed by a single index pattern.

Author: I've applied your suggestion, but I also used `*` around "Discover", like `*Discover*` as I've Discover been referenced like that in other places.

Also, instead of "you use most often", I've changed it to "likely used in the visualization", because those documents are not often used ones but the ones that were used to construct the visualization, plus, potentially some extra documents.

you to *Discover* narrowing down documents to ones likely used in
the visualization.

"Explore underlying data" actions are available for visualizations which are
backed by a single index pattern.

> There are two "Explore underlying data" actions: (1) one which appears in the

Reviewer: Suggest:

> You can access **Explore underlying data** in two ways: from the panel context
> menu or from the menu that appears when you mouse over the chart.

Author: Applied your suggestion, but instead of "...when you mouse over the chart" used "...when you interact with the chart".

panel's context menu; (2) and the other which appears when user clicks or brushes
the chart.

[float]
[[explore-data-from-panel-context-menu]]
=== Explore data from panel context menu

> Context menu "Explore underlying data" action will navigate you to *Discover*,

Reviewer 2: I suggest changing this a bit to say:

> Context menu "Explore underlying data" action will redirect you to *Discover*, and will carry over the index pattern, all filters, query, and time range applied to the visualization.

Reviewer: Suggest:

> The **Explore underlying data** action in the panel menu navigates you to Discover,
> carrying over the index pattern, filters, query, and time range for the visualization.

Author: Done.

to the index pattern used in the visualization and will carry over all filters,
query, and time range applied to the visualization.

[role="screenshot"]
image::images/explore_data_context_menu.png[Explore underlying data from panel context menu]

[float]
[[explore-data-from-chart]]
=== Explore data from chart action

> In-chart "Explore underlying data" action will also navigate you to *Discover*,

Reviewer 2: I would re-phrase this a bit to:
> In-chart "Explore underlying data" action will also redirect you to *Discover*, and will carry over the index pattern, all filters, query, and time range applied to the visualization, but it will also apply the filters and time range created from the click or brush event that triggered this action.

Reviewer: Initiating **Explore underlying data** from the chart also navigates to Discover,
carrying over the current context.  In addition, this action applies the filters and time range
created by the events that triggered the action.

Reviewer: Suggest:

> Initiating **Explore underlying data** from the chart also navigates to Discover,
> carrying over the current context for the visualization.  In addition, this action applies the filters and time range
> created by the events that triggered the action.

Author: Done.

to the index pattern used in the visualization and will carry over all
filters, query and time range, but it will also apply the filters and time range

> created from the click or brush events that triggered this action. For example,

Reviewer 2: I think it may be better to reference a specific example here, as illustrated by the image below. I.e. describe what would happen, if the user selected this action after clicking on that specific slice of the chart.

Reviewer: ++

Author: I've replaced this paragraph with the one Gail suggested above. I think it is good now, tell me if you think we should still provide an example.

if you will brush a chart the time range associated
with that brush event will be carried over to *Discover*.

[role="screenshot"]
image::images/explore_data_in_chart.png[Explore underlying data from chart]

> In-chart action can be optionally disabled by adding below line to your `kibana.yml` config.

Reviewer: ```suggestion
You can disable this action by adding the following line to your `kibana.yml` config.
```

Author: Done.
